packages/terra-open-source-scripts/src/terra-cli/git-tag-mono-repo/gitTagMonoRepoHandler.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
```

Comment:
I'd like to flex these scripts automatically based on whether the current repo is a monorepo

Reply:
This is already being done?